python/ml4ir/base/data/ranklib_to_ml4ir.py

```diff
@@ -0,0 +1,98 @@
+import pandas as pd
+import argparse
+import logging
+import numpy as np
+
+
+#Constants for the column names given to the query id and graded relevance.
+QUERY_ID_NAME = 'qid'
+QUERY_TYPE = 'str'
+RELEVANCE_NAME = 'relevance'
```

Comment:
@mohazahran You should accept all these 3 constants as command line arguments with these values as defaults. That is the best middle ground.

Reply:
I can read the names from the feature_config. What do you think?